Jenkinsfile

```diff
@@ -96,7 +96,7 @@ pipeline {
         stage('Stable Tests') {
             steps {
                 echo 'Running tests'
-                sh 'mvn -B -e -fae test ${MVN_SHOW_TIMESTAMPS} -P ci-test ${MVN_LOCAL_REPO_OPT} -Dassembly.skipAssembly=true jacoco:report-aggregate@jacoco-report'
+                sh 'mvn -B -e -fae clean test ${MVN_SHOW_TIMESTAMPS} -P ci-test ${MVN_LOCAL_REPO_OPT} -Dassembly.skipAssembly=true jacoco:report-aggregate@jacoco-report'
```

Comment:
clean is required for storing results in the build cache 
the remote build cache will ensure we don't spend to much rebuilding already built stuff

anyway build time is negligible compared to test runtime so event with suboptimal caching we should benefit